src/api/structure/fixture.js

```diff
@@ -14,6 +15,7 @@ export default class Fixture extends TestingUnit {
     constructor (testFile) {
         super(testFile, 'fixture');
 
+        this.id   = nanoid(7);
```

Comment:
I propose to use the `hammerhead.generateUniqueId` function to generate fixture identifier instead of `nanoid`.
The `hammerhead.generateUniqueId` creates the URL friendly id. It allows us to use them as navigation identifier in the Dashboard web interface.

Reply:
Does we need an unique identifier for test? If it' s true, we can generate it on the TestingUnit class level?

Reply:
> Does we need an unique identifier for test? If it' s true, we can generate it on the TestingUnit class level.

Let address this question to @aleks-pro

Reply:
For me, it's a good idea to create and id for TestingUnit base class